doc/Advanced-configuration.md

```diff
@@ -351,6 +351,12 @@ There are some additional options that influence all database connections in the
     * **Syntax:** `{replaced_wait_timeout, TimeInMilliseconds}`
     * **Default:** `2000`
 
+* **cowboy_server_name** (local)
+    * **Description:** If configured, replaces Cowboy's default name returned in `server` HTTP response header. It may be use for extra security, as it makes harder for the malicious user harder to learn what HTTP software is running under specific port. This option applies to **all** listeners started by `ejabberd_cowboy` module.
```

Comment:
* **Description:** If configured, replaces Cowboy's default name returned in the `server` HTTP response header. It may be used for extra security, as it makes it harder for the malicious user to learn what HTTP software is running a under specific port. This option applies to **all** listeners started by the `ejabberd_cowboy` module.